examples/pvtol-lqr.py

```diff
@@ -21,35 +20,37 @@
 #
 
 # System parameters
-m = 4;				# mass of aircraft
-J = 0.0475;			# inertia around pitch axis
-r = 0.25;			# distance to center of force
-g = 9.8;			# gravitational constant
-c = 0.05;	 		# damping factor (estimated)
+m = 4  # mass of aircraft
+J = 0.0475  # inertia around pitch axis
+r = 0.25  # distance to center of force
+g = 9.8  # gravitational constant
+c = 0.05  # damping factor (estimated)
 
 # State space dynamics
-xe = [0, 0, 0, 0, 0, 0];        # equilibrium point of interest
-ue = [0, m*g];                  # (note these are lists, not matrices)
+xe = [0, 0, 0, 0, 0, 0]  # equilibrium point of interest
+ue = [0, m * g]  # (note these are lists, not matrices)
 
 # Dynamics matrix (use matrix type so that * works for multiplication)
-A = matrix(
-    [[ 0,    0,    0,    1,    0,    0],
-     [ 0,    0,    0,    0,    1,    0],
-     [ 0,    0,    0,    0,    0,    1],
-     [ 0, 0, (-ue[0]*sin(xe[2]) - ue[1]*cos(xe[2]))/m, -c/m, 0, 0],
-     [ 0, 0, (ue[0]*cos(xe[2]) - ue[1]*sin(xe[2]))/m, 0, -c/m, 0],
-     [ 0,    0,    0,    0,    0,    0 ]])
+A = np.matrix(
+    [[0, 0, 0, 1, 0, 0],
+     [0, 0, 0, 0, 1, 0],
+     [0, 0, 0, 0, 0, 1],
+     [0, 0, (-ue[0] * np.sin(xe[2]) - ue[1] * np.cos(xe[2])) / m, -c / m, 0, 0],
+     [0, 0, (ue[0] * np.cos(xe[2]) - ue[1] * np.sin(xe[2])) / m, 0, -c / m, 0],
+     [0, 0, 0, 0, 0, 0]]
+)
 
 # Input matrix
-B = matrix(
+B = np.matrix(
```

Comment:
Since the Numpy `matrix` class is being deprecated, it might make more sense to use `np.array` here, which should still work and won't require changes later.

Note, however, that if we make that change then we can no longer use `*` for matrix multiplication.  We have to use `dot` or `@` (python3 only).

Reply:
I can make that change now if you want.  I wasn't sure whether now is the right time.

Reply:
It's a good question about whether to do this now or later.  There is some discussion about how we are going proceed in issue #233 and PR #292.  Since this is existing (as opposed to new) code, it would be OK to leave.

On the other hand, it can be annoying to get warning message when you run an example (which is presumably what happens here, since Numpy is putting out DeprecationPending warnings whenever a new `matrix` object is created).

You are the author of the PR, so in the end you get to decide -:).

Reply:
Turns out converting from `np.matrix` to `np.array` is not going to be that easy in all cases.  The problem with np.array is that when a result is one-dimensional (vector) it tends to reduce the result to shape (n, ) whereas matrices always remain 2-dimensional.

I made the conversion for two of the scripts but `pvtol-lqr.py` is going to take a bit more of a time investment so I propose leaving that for another day.

Reply:
Agree, the updates are a bit of a pain (which is why we are delaying things in PR #292, in fact).  Seems fine to leave alone and we can handle them at a later date.

Reply:
If this final push passes the tests then I think it's all ready to be merged.  I put some TODO comments in `pvtol-lqr.py` to help with the future conversion from array -> matrix.